### DISCUSSION TITLE
Old system bugging out

### DISCUSSION THREAD

**Discussion by Edward-chin2102** [valinet/ExplorerPatcher#1576] [Q&A] [answered]
I deleted the newest version and then accidentally reinstalled an older version (I don’t know which). My laptop is now seemingly restarting the file explorer etc endlessly and I can’t access anything else but the task manager. How can I uninstall the old version from the task manager?

**Comment by pyrates999** [comment, marked as answer]
Boot into safe mode and try uninstalling it: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

**Reply by Edward-chin2102** [reply]
Thanks a bunch!! I’m not very technically savy but it worked ☺️
